repository_service_tuf_api/bootstrap.py

```diff
@@ -158,6 +136,17 @@ def post_bootstrap(payload: BootstrapPayload) -> BootstrapPostResponse:
         queue="metadata_repository",
         acks_late=True,
     )
+    logging.info(f"Bootstrap task {task_id} sent")
+
+    # start a thread to check the bootstrap process
+    Thread(
+        None,
+        _check_bootstrap_status,
+        kwargs={
+            "task_id": task_id,
+            "timeout": settings.get("BOOTSTRAP_TIMEOUT", 300),
```

Comment:
Do we want to document this as an option or it will be an internal one?

Reply:
The better option is not creating an internal one but adding to the bootstrap payload as an optional field.
It will give tho the user easier and more flexible usage. Imagine, for example, a repository that chooses to use 16k delegated hash. The bootstrap will take a while, depending o their computing hardware.